src/tasks/tasks.service.ts

```diff
@@ -22,4 +23,9 @@ export class TasksService {
   async findAllTasks(): Promise<Task[]> {
     return await this.tasksRepository.findAll();
   }
+
+  async updateTask(id: number, taskData: UpdateTaskDto): Promise<Task> {
```

Comment:
```suggestion
  async updateTask(id: number, updateTaskDto: UpdateTaskDto): Promise<Task> {
```

Let's name the param consistently with the type.